.golangci.yml

```diff
@@ -1,97 +1,106 @@
+version: "2"
 linters:
   # Don't enable any linters by default; just the ones that we explicitly
   # enable in the list below.
-  disable-all: true
+  default: none
   enable:
     - bidichk
-    - gofmt
-    - goimports
     - govet
     - misspell
     - revive
-
-# Configuration for how we run golangci-lint
-run:
-  timeout: 5m
```

Comment:
is the timeout option still available?  If so, we might want to retain that.  Alternatively, we could enforce it within the github action also, which might even be better.

Reply:
Good callout! Think I misread the migration guide, I saw: 

> run.timeout: the existing value is ignored because, in v2, there is no timeout by default.

And thought that they had removed the timeout, but fairly certain they just adjusted the default and had the migration tool reset to this default (e.g., no timeout) instead of preserving the value. 

Will adjust!

Reply:
> And thought that they had removed the timeout, but fairly certain they just adjusted the default and had the migration tool reset to this default (e.g., no timeout) instead of preserving the value.

Yup this does seem to be the case, example in https://golangci-lint.run/usage/configuration/#run-configuration shows setting this to 5m (which I assume was the previous default). Have added this back in.

Thanks again for flagging this!